pkg/handlers/user/props.go

```diff
@@ -53,7 +53,12 @@ func (cmd *GetCmd) Run(c *cc.CommonCtx) error {
 		if tokenErr != nil {
 			return tokenErr
 		}
-		return jsonx.OutputJSON(c.StdOut(), token)
+		tokenBytes, err := json.Marshal(token)
```

Comment:
This change breaks sending the result to stdout.

```
aserto3 user get token > token.json
```

Instead it goes to stderr

```
aserto3 user get token &> token.json
```